examples/files_example/public/plugin.ts

```diff
@@ -18,7 +18,10 @@ export class FilesExamplePlugin
     core: CoreSetup<FilesExamplePluginsStart>,
     { files, developerExamples }: FilesExamplePluginsSetup
   ) {
-    files.registerFileKind(exampleFileKind);
+    files.registerFileKind({
+      id: exampleFileKind.id,
+      allowedMimeTypes: exampleFileKind.allowedMimeTypes,
```

Comment:
Would be nice to still set `maxSizeBytes` to some value here to include that in our example.